ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -316,7 +314,13 @@ object CommandService {
       message: String,
   ) extends Error
   final case class GrpcError(status: io.grpc.Status) extends Error
-  final case class InternalError(id: Option[Symbol], message: String) extends Error
+  final case class InternalError(id: Option[Symbol], error: Throwable \/ String) extends Error
```

Comment:
Is the throwable used in any way later? It would seem simpler to just take the String along.